src/accessories/CampaignImporterExporter.ttslua

```diff
@@ -47,23 +47,12 @@ local campaignTokenData = {
 local COLORS = { "White", "Orange", "Green", "Red" }
 
 function onLoad()
-  self.createButton({
-    click_function = "findCampaignFromToken",
-    function_owner = self,
-    label = "Import",
-    tooltip = "Load in a campaign save from a token!\n\n(Token can be anywhere on the table, but ensure there is only 1!)",
-    position = { x = -1, y = 0.2, z = 0 },
-    font_size = 400,
-    width = 1400,
-    height = 600,
-    scale = { 0.5, 1, 0.5 },
-  })
   self.createButton({
     click_function = "createCampaignToken",
     function_owner = self,
     label = "Export",
-    tooltip = "Create a campaign save token!\n\n(Ensure all chaos tokens have been unsealed!)",
-    position = { x = 1, y = 0.2, z = 0 },
+    tooltip = "Create a campaign save token!",
+    position = { x = -1, y = 0.2, z = 0 },
```

Comment:
Nitpick: The "y"-value should be 0.11 since the thickness is 0.2, but this is calculated from the center. (0.10 will cause flickering since the asset and the button will "fight" for the same position)

Reply:
Assuming you meant .21, but yes!

Reply:
Nope, I did mean 0.11 😅